zerver/tests/test_hooks.py

```diff
@@ -1181,3 +1181,13 @@ def test_circleci_build_in_success_status_when_previous_build_failed_too(self):
         expected_subject = u"RepoName"
         expected_message = u"[Build](https://circleci.com/gh/username/project/build_number) triggered by username on master branch fixed."
         self.send_and_test_stream_message('build_passed_when_previous_build_failed', expected_subject, expected_message)
+
+class CrashlyticsHookTests(WebhookTestCase):
+    STREAM_NAME = 'crashlytics'
+    URL_TEMPLATE = "/api/v1/external/crashlytics?stream={stream}&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'crashlytics'
+
+    def test_circleci_build_in_success_status(self):
+        expected_subject = u"Issue report"
+        expected_message = u"Issue [Issue Title](http://crashlytics.com/full/url/to/issue) occurred."
```

Comment:
I think the body for these may need to have some sort of mapping based on the event and payload_type fields; e.g. the example event you have above is an impact level change.